staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -50,7 +50,6 @@ var baseOpts = []VersionedOptions{
 		// options should always be present.
 		IntroducedVersion: version.MajorMinor(1, 0),
 		EnvOptions: []cel.EnvOption{
-			cel.HomogeneousAggregateLiterals(),
```

Comment:
It's okay to leave this option enabled as it won't conflict with the new call. You can also remove it, but I thought I'd mention the compatibility of the two options

Reply:
Thanks for mentioning it! We could go with enable with `cel.HomogeneousAggregateLiterals(),` without adding it in new validator as well. Curious if there is preference on either direction :)

Reply:
I think just preserving the old option here is okay since it gives you a bit more reassurance about how things are behaving between versions. Going forward, this option should be deprecated in favor of the `ValidateHomogeneousAggregateLiterals` option (what a mouthful)